WalletWasabi.Fluent/Helpers/ConfirmationTimeLabel.cs

```diff
@@ -19,6 +19,6 @@ public static string SliderLabel(TimeSpan timeSpan)
 			return "fastest";
 		}
 
-		return "~" + TimeSpanFormatter.Format(timeSpan, new TimeSpanFormatter.Configuration(" days", " hours", " minutes"));
+		return "~" + TimeSpanFormatter.Format(timeSpan, new TimeSpanFormatter.Configuration(" day", " hour", " min"));
```

Comment:
```suggestion
		return "~" + TimeSpanFormatter.Format(timeSpan, new TimeSpanFormatter.Configuration(" day", " hour", " minute"));
```
It's not much longer...

Reply:
I did that for consistency as the short version is used on the chart.